src/App.js

```diff
@@ -1,45 +1,77 @@
 import './styles/reset.css'
 import './styles/index.css'
-
+import { useState } from 'react'
 import initialStoreItems from './store-items'
 
-/*
-Here's what a store item should look like
-{
-  id: '001-beetroot',
-  name: 'beetroot',
-  price: 0.35
-}
-
-What should a cart item look like? 🤔
-*/
+export default function App () {
+  // Setup state here...
+  const [store, setStore] = useState(initialStoreItems)
+  const [cart, setCart] = useState([])
 
-console.log(initialStoreItems)
+  function addToCart (item) {
+    checkQuantity(item.id) ? IncQuantity(item) : setCart([...cart, { ...item }])
```

Comment:
like the use of the ternary operator. everyone seems to use it apart from me! Need to look into this